### PR TITLE
ic-status-tag - Added Prop which conditionally makes the comp an aria-live region

### PR DESCRIPTION
**Pull request by gd2910**
## Summary of the changes
The announced tag adds the role='status' prop and adds the aria-live status, more explicit aria-label descriptors added to span element. A11y tests changed to reflect and unit tests added for new prop.

## Related issue
#20  

## Checklist
- [x] I have added relevant unit and visual regression tests.
- [ ] I have manually accessibility tested any changes, if relevant.
- [x] I have ensured any changes match the Figma component library.

### CODE REVIEW

**Review by GCHQ-Developer-530 — COMMENTED** [review]
Think the branch just needs to rebase off develop to get rid of the other 5 commits, but looks good other than that :)

File: packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ export class StatusTag {
    * If `true`, the small styling will be applied to the status tag.
    */
   @Prop() small?: boolean = false;
+  /**
+   * If 'true' the component will act as an 'aria-live' region.
```
Comment (ad9242) [inline packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx:31]:
```suggestion
   * If 'true', the component will act as an 'aria-live' region.
```

**Review by ad9242 — CHANGES_REQUESTED** [review]
minor typo, but looks good otherwise

**Review by GCHQ-Developer-530 — DISMISSED** [review]

**Comment by gd2910** [comment]
PR closed due to git issues, please see new status-tag PR

**Pull request closed without merging**